### ISSUE TITLE
Prettier error traps parallel-prettier in infinite loop

### ISSUE THREAD

**Issue by thomasaull** [microsoft/parallel-prettier#5]
I'm using this package to format a folder of generated markup. However, if the markup contains an error, for example an opening tag which is not closed

```html
<div class="myClass">
  <div class="myClass_inner">
  <a href="#">Link</a>
</div>
```
it would run forever if not stopped manually. The original prettier throws an error, continues to format the other files and returns an exit code > 0. 

In my mind, this should be an easy fix, since prettier already spits out the error/exit code, however I don't have much experience with gulp in general.

I'd be happy to provide a test repo or try to fix this issue if I can get some pointers in the right direction.

**Comment by connor4312** [comment]
Fix published in 2.0.1. Sorry for the delay.

**Comment by thomasaull** [comment]
Thanks @connor4312 👍
